### ISSUE TITLE
potential problem with V / VC phonotactic

### ISSUE THREAD

**Issue by masterkrang** [masterkrang/kako#9]
Imagine a scenario where the vowel e = our (as in 'our train is arriving' or 'this land is our land').

If we want to reuse this word to construct a new word, for example let's say a word 'ke' = 'planet'. imagine we wanted to use the word 'our' (e) and 'planet' (ke) together to form a new word 'eke' meaning 'Earth'. This word would mean 'earth', since Earth is "our planet". 

Here is the problem, if we allow single sounds (vowels) to make up root words, then we would be in theory defying the V / VC structure, since the word 'eke' (earth) would be a VCV structure. 

Possible solutions:

1 allow this to happen, simply view words that are composed of one vowel to be reusable across the language.
2 do not allow this to happen, vowels that comprised words can only live on their own
3 do not allow vowels to be words on their own, only allow them to be used in conjunction with consonants. (so only CV structure would be allowed for root words).

As of now I think I'm going for option 1, but it seems like it could have unforeseen issues later, and actually introduces complications. But, it is more reusable and powerful, which is desirable. 

Thoughts?
cc @yhdaphne @JoseUgalde

**Comment by ghost** [comment]
"Here is the problem, if we allow single sounds (vowels) to make up root words, then we would be in theory defying the V / CV structure, since the word 'eke' (earth) would be a VCV structure." 

=> I assume you meant CV structure (?) since ke for 'planet' is in a consonant-vowel structure (hence CV).

For your question, there is no problem in adopting V/CV structure. Allowing only V/CV structure makes the word eke 'earth' a two-syllable structure, in which e.ke is composed by the first V-structure and the second CV-structure. Given this understanding could work, the question is how many syllables in a word you want to allow it in Kako. This is maybe a phonotactic question.

**Comment by JoseUgalde** [comment]
I am not sure (we are gonna have to revisit this issue in the future) about such a limited inventory of syllables.Are you familiar with the concept of phonetic redundancy?A certain degree of redundancy is required because some phonemes are very similar...

Date: Tue, 20 Oct 2015 01:46:13 -0700

**Comment by masterkrang** [comment]
@yhdaphne good point. i guess the concern is what if we move to a different planet like mars and we want to call it "our planet two". so imagine the word for "two" is "ne". so then we would have a word like "ekene". maybe this isn't a problem but worth thinking about. Just want to avoid a situation like in English where there are too many consonants and vowels mixed up.

**Comment by masterkrang** [comment]
@JoseUgalde i was also thinking of something similar to "phonetic redundancy". for example, Chinese has 5 different ways to tonaly say "ma", but there are 28 different words that are said using the "ma" sound. 

I'm not sure we are talking about the same thing though so can you link me?

I am concerned that if there are too little sounds then a person may get confused as sound repeat, or as they need to repeat and take on different meanings. At that point we would only be able to allow context to determine the meaning of the word, which could be troublesome. 

I might try to research how man root words we need by comparing world language root words, if it's possible.
